src/js/burials/components/IntroductionPage.jsx

```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
   render() {
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply online for burial benefits"/>
-        <p>
-          Fill out this application with the most accurate information you have. The more accurate it is, the more likely you are to get a rapid response.
-        </p>
-        <p>
-          VA uses the information you submit to determine your eligibility and to provide you with the best service.
-        </p>
-        <p>
-          Federal law provides criminal penalties, including a fine and/or imprisonment for up to 5 years, for concealing a material fact or making a materially false statement. (See <a href="https://www.justice.gov/usam/criminal-resource-manual-903-false-statements-concealment-18-usc-1001" target="_blank">18 U.S.C. 1001</a>)
-        </p>
+        <FormTitle title="Apply for burial benefits"/>
+        <p>This application is equivalent to Form 21P-530 (Application for Burial Benefits).</p>
+        <SaveInProgressIntro
+            pageList={this.props.route.pageList}
+            resumeOnly
+            {...this.props.saveInProgressActions}
+            {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Veteran’s Social Security number or VA file number (required)</li>
+                <li>Veteran’s military discharge papers (DD214 or other separation documents)</li>
```

Comment:
Does the fact that military history is [not required](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3610#issuecomment-312363978) for this form mean that it should not be listed here? Or that it should not have the word required after it?